documentation/modules/auxiliary/scanner/http/crawler.md

```diff
@@ -0,0 +1,98 @@
+## Description
+
+This module is a http crawler, it will browse the links recursively from the web site. If you have loaded a database plugin and connected to a database this module will report web pages and web forms.
+
+## Vulnerable Application
+
+You can use any web application to test the crawler 
+
+## Options
+
+  **URI**
+
+  Default path is /
+
+  **DirBust**
+
+  Bruteforce common url path, default is `true` but may generate noise in reports.
+
+  **HttpPassword/HttpUsername/HTTPAdditionalHeaders/HTTPCookie**
+
+  You can add some login information
+
+  **UserAgent**
+
+  Default User Agent is `Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)` 
```

Comment:
Could we link to some place that has additional ones?  Not sure if MSF/R7 has a list, but the more reputable site the better.